File: src/main/java/com/itsci/mju/maebanjumpen/repository/PenaltyRepository.java
package com.itsci.mju.maebanjumpen.repository;

import com.itsci.mju.maebanjumpen.model.Penalty;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface PenaltyRepository extends JpaRepository<Penalty, Integer> {
}

File: src/main/kotlin/com/itsci/mju/maebanjumpen/penalty/repository/PenaltyRepository.kt
package com.itsci.mju.maebanjumpen.penalty.repository

import com.itsci.mju.maebanjumpen.entity.Penalty
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface PenaltyRepository : JpaRepository<Penalty, Int>

